CONTRIBUTING.md

```diff
@@ -1,27 +1,146 @@
-"consider repository-name" (in lower case) with a link to the
-repository as the issue comment.
-
-## Guidelines
-* Search previous suggestions before making a new one, as yours may be a duplicate.
-* Add only one link per Pull Request.
-* Add the link: `* [project-name](https://github.com/usnistgov/REPOSITORY-NAME) - A short description ends with a period.`
-    * Keep descriptions concise.
-* Add a section if needed.
-    * Add the section description.
-    * Add the section title to Table of Contents.
-* Write `Closes #__` in the Pull Request description, referring to the associated Issue.
-* Fill out the Pull Request checklist as completely as possible. It should take 10 minutes or less to complete.
+
+To nominate a repository, file an [Issue][_issu] labeled "consider
+repository-name" (in lower case) with a link in the comment field.
+
+It would help peer review of the repository if you also paste the
+[checklist](AWESOME.md) into the comment field, and check off
+whatever applies. Don't worry if several boxes go unchecked:
+it gives the developers room to grow!
+
+## Guidelines for Contributors
+
+To add a repository to the curated list, please
+
+* Locate the [Issue][_issu] in which the repository was nominated,
+  or create a new Issue if one does not already exist.
+* Open [README.md][README.md] and add the link in a single line:
+  `* [project-name](url) - A short description ends with a period.`
+  Add only one link per Pull Request.
+* Create a [Pull Request][_plrq] from your modified [README.md](README.md).
+* Write `Closes #__` in the Pull Request description, referring to the
+  associated Issue.
+* Paste in and complete the [checklist](AWESOME.md). It should take
+  10 minutes or less to complete.
 * Don't mention `NIST` in the description as it's implied.
 * Check your spelling and grammar. Remove any trailing whitespace.
-* Send a Pull Request with the reason why the library is awesome.
+
+The original [awesome.re][_awre] has more detailed instructions.
+
+## Guidelines for Reviewers
+
+Each Pull Request must be approved by at least one reviewer before merging.
+If the repository is awesome, the review process should be painless. If it
+misses the bar, reviewers are asked to provide friendly, clear feedback on
+how to improve. If you (as a reviewer) file issues against the nominated
+repository, please include links to those issues in your review.
+
+While "awesome" is a subjective evaluation, there is a bar that all repositories
+must clear. Please ensure that the prospective repository is fundamentally sound.
+
+### Required Files
+
+Developers signed the [Rules of Behavior][_grob], which requires both
+LICENSE.md (or similar) and README.md (or similar).
+
+#### LICENSE.md
+
+LICENSE.md must contain the official [dedication to the public domain][_pub].
+Assignments or reservations of copyright, *e.g.* under [BSD][_bsd], [GPL][_gpl],
+or [MIT][_mit] licensing terms, are unlawful per [17 U.S.C. §105][_copy].
```

Comment:
- Projects done in collaboration with external partners? 
 - Projects conducted by non-Federal staff, e.g., contractors, FGRs, etc.?